mkdocs.yml

```diff
@@ -14,29 +14,23 @@ pages:
   - 'Quick Start Guide': 'quick_start_guide_new.md'
   - 'Installation and Administration manual':
       - 'Introduction': 'installation_and_administration_manual/introduction.md'
-      - 'Batch processing: Some words about Cosmos and its ecosystem': 'installation_and_administration_manual/batch/some_words_about_cosmos_and_ecosystem.md'
-      - 'Batch processing: HAAS engine (Shara version)': 'installation_and_administration_manual/batch/haas_engine_sahara.md'
-      - 'Batch processing: HAAS engine (shared Hadoop version)': 'installation_and_administration_manual/batch/haas_engine_shared_hadoop.md'
-      - 'Batch processing: Storage cluster': 'installation_and_administration_manual/batch/storage_cluster.md'
-      - 'Batch processing: Cosmos GUI': 'installation_and_administration_manual/batch/cosmos_gui.md'
-      - 'Batch processing: OAuth2-protected APIs': 'installation_and_administration_manual/batch/oauth2_protected_apis.md'
-      - 'Batch processing: Tidoop': 'installation_and_administration_manual/batch/tidoop.md'
-      - 'Batch processing: Cygnus': 'installation_and_administration_manual/batch/cygnus.md'
-      - 'Batch processing: Sanity Check Procedures': 'installation_and_administration_manual/batch/sanity_check_procedures.md'
-      - 'Batch processing: Diagnosis Procedures': 'installation_and_administration_manual/batch/diagnosis_procedures.md'
-      - 'Stream processing: Sinfonier Architecture Overview': 'installation_and_administration_manual/streaming/stream_processing_sinfonier_architecture.md'
-      - 'Stream processing: Sinfonier Drawer': 'installation_and_administration_manual/streaming/stream_processing_sinfonier_drawer.md'
-      - 'Stream processing: Sinfonier API': 'installation_and_administration_manual/streaming/stream_processing_sinfonier_api.md'
-      - 'Stream processing: Sinfonier BackEnd': 'installation_and_administration_manual/streaming/stream_processing_sinfonier_backend.md'
-      - 'Annex A: Creating a public-private key pair and installing the public key': 'installation_and_administration_manual/batch/annex_a_create_and_install_keypair.md'
-      - 'Annex B: Get a keystone auth token': 'installation_and_administration_manual/batch/annex_b_get_keystone_auth_token.md'
-      - 'Annex C: Deploy Apache Storm Cluster': 'installation_and_administration_manual/streaming/annex_c_deploy_storm_cluster.md'
+      - 'Some words about Cosmos and its ecosystem': 'installation_and_administration_manual/some_words_about_cosmos_and_ecosystem.md'
+      - 'HAAS engine (Shara version)': 'installation_and_administration_manual/haas_engine_sahara.md'
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cosmos/pull/214/commits/a37133d55f651e5407b3eb87fb9192bc1a85c89e